packages/website/ts/blockchain.ts

```diff
@@ -682,8 +682,7 @@ export class Blockchain {
                     // Note: it's not entirely clear from the documentation which
                     // errors will be thrown by `watch`. For now, let's log the error
                     // to rollbar and stop watching when one occurs
-                    // tslint:disable-next-line:no-floating-promises
-                    errorReporter.reportAsync(err); // fire and forget
+                    errorReporter.report(err); // fire and forget
```

Comment:
Remove comment. This is no longer fire and forget if it's synchronous.

Reply:
It is fire and forget :P